builtin/logical/transit/backend.go

```diff
@@ -231,29 +231,37 @@ func (b *backend) autoRotateKeys(ctx context.Context, req *logical.Request) erro
 			continue
 		}
 
-		// If the policy's automatic rotation interval is 0, it should not
-		// automatically rotate.
-		if p.AutoRotateInterval == 0 {
+		err = b.rotateIfRequired(ctx, req, key, p)
+		if err != nil {
+			errs = multierror.Append(errs, err)
 			continue
```

Comment:
trivial nit: You don't need the continue here.

Reply:
Good eye; removed!